src/views/Markets/components/Overview.tsx

```diff
@@ -146,7 +149,13 @@ export const Overview = () => {
 							>
 								{healthFactor &&
 									(healthFactor.isFinite() ? (
-										healthFactor.toFixed(2)
+										healthFactor.isLessThanOrEqualTo(0) ? (
+											'-'
+										) : healthFactor.gt(10000) ? (
+											'> 10000'
```

Comment:
Maybe add a tooltip so that users with `> 10000` health factors can hover to see the full value?